config/eslintrc.base.js

```diff
@@ -38,7 +38,8 @@ module.exports = {
     'no-console': [1, { allow: ['warn', 'error', 'info'] }],
     'no-shadow': 0,
     'no-use-before-define': 0,
-    'no-cond-assign': [2, 'except-parens']
+    'no-cond-assign': [2, 'except-parens'],
+    'no-multi-str': 0
```

Comment:
Surprised that it is 1 by default.

Reply:
Yeah, it’s referring to old browsers which don’t support it (pre-ES5), so … I think this is very safe to disable :)